src/neo-vm/Helper.cs

```diff
@@ -27,18 +28,31 @@ public static ulong ReadVarInt(this BinaryReader reader, ulong max = ulong.MaxVa
             return value;
         }
 
-        public static string ReadVarString(this BinaryReader reader)
+        internal static string ReadVarString(this BinaryReader reader)
         {
             return Encoding.UTF8.GetString(reader.ReadVarBytes());
         }
 
-        public static void WriteVarBytes(this BinaryWriter writer, byte[] value)
+        public static uint ToInteropMethodHash(this string method)
+        {
+            return ToInteropMethodHash(Encoding.ASCII.GetBytes(method));
+        }
+
+        public static uint ToInteropMethodHash(this byte[] method)
+        {
+            using (SHA256 sha = SHA256.Create())
```

Comment:
Why not use a faster hash function? Do we really need such a powerful hash for this purpose?

Reply:
That's a fair point, but since Neo needs to implement it anyway, it's better to reuse that as much as possible. We hope to propose accelerations to this and other functions in the next months, so that won't be an issue.

Reply:
Ahh! @jsolman , we will usually never need to execute that on Runtime xD when compiling the code, neo-compiler will translate interop names directly to 4 bytes, what I believe everyone will use because it will drastically reduce avm size. So, this line will only be used for compatibility reasons ;)